cmake/QSkinnyConfig.cmake

```diff
@@ -1,2 +1 @@
 include("${CMAKE_CURRENT_LIST_DIR}/QSkinnyTargets.cmake")
-include("${CMAKE_CURRENT_LIST_DIR}/QskTools.cmake")
```

Comment:
We could use the `OPTIONAL` keyword to disable errors and express it being optional

> If OPTIONAL is present, then no error is raised if the file does not exist.
>
> -- https://cmake.org/cmake/help/latest/command/include.html

Reply:
Addressed in #359